harvestingkit/elsevier_package.py

```diff
@@ -295,7 +295,7 @@ def _normalize_article_dir_with_dtd(self, path):
     def _add_references(self, xml_doc, rec, refextract_callback=None):
         for label, authors, doi, issue, page, title, volume, year,\
```

Comment:
that's a lot of values to unpack by position and therefore error-prone and dangerous in a subtle way. using a data structure with labels like named tuples or dict would make this clearer and more robust